ios/PurchasesHybridCommon/ObjCAPITester/RCHybridCommonAPITest.m

```diff
@@ -39,7 +40,7 @@ - (void)testAPI {
                                                               RCErrorContainer * _Nullable error) {
     }];
 
-    [RCCommonFunctionality appUserID];
+    NSString *appUserID __unused = [RCCommonFunctionality appUserID];
```

Comment:
for some reason, the API testers actually work _better_ when testing against swift types, and catch some stuff that wouldn't be caught in the objc one. this `__unused` param is one of them